cmd/node/config.go

```diff
@@ -50,3 +52,16 @@ func (cfg *NodeConfig) allFieldsPresent() error {
 	}
 	return nil
 }
+
+func (config *NodeConfig) generateNodeID() (*[]byte, error) {
+	publicKey, err := ioutil.ReadFile(config.Keys.PathToPublic) //nolint:gosec // no user input for filename
+	if err != nil {
+		return &[]byte{}, err
+	}
+	buf := sha1.Sum(publicKey) //nolint:gosec //public key is not so secured
+	mHashBuf, err := multihash.EncodeName(buf[:], "sha1")
+	if err != nil {
+		return &[]byte{}, err
```

Comment:
@stepanvalyavskiy I don't think we need to allocate empty slice for error return